hugo/content/docs/tracing.md

```diff
@@ -10,148 +10,128 @@ title: Distributed Tracing
+[Realtime map is using Proto.OpenTelemetry](https://github.com/asynkron/realtimemap-dotnet/blob/main/Backend/Program.cs#L19) and might be used as a working example.
 
-In Proto.OpenTracing you can create a local tracer or use a global tracer instance. Tracer allows to create scopes for send, receive, request, forward message functions. Depending on the function, a Span is created, which is marked with different tags, such as `MessageType`, `TargetPID`, `SenderPID`, `ActorPID`, `ActorType`. Proto.OpenTracing allows to log errors, create custom Spans, and work with baggage. Let’s look at all these features in more detail.
+First thing that needs to be done is registration of [TracerProvider](https://opentelemetry.io/docs/reference/specification/trace/api/#tracerprovider). We can use extension placed in the package `OpenTelemetry.Extensions.Hosting`.
+This helps us to register `TracerProvider` with configured instance of `TracerProviderBuilder`. In this place we can put some common instrumentation that is applied to all Spans.
+It is also possible to setup OpenTelemetry Protocol exporter depending which backend is used. In order to reuse below example, `OpenTelemetry.Exporter.OpenTelemetryProtocol` nuget package is needed.
 
-### Setup tracing for Proto.Actor
+Little remark, C# OpenTelemetry implementation is based on [System.Diagnostics.Activity](https://docs.microsoft.com/en-us/dotnet/core/diagnostics/distributed-tracing-instrumentation-walkthroughs).
+Naming convention placed there is not equivalent with OpenTelemetry specification. From that reason, in the C# implementation Span is named as Activity.
 
-In order to use Proto.OpenTracing with Proto.Actor first you need to create Proto.Actor and call extension method `WithOpenTracing()`.
+`AddProtoActorInstrumentation()` extension from `Proto.OpenTelemetry` package, is adding Proto.Actor [activity source](https://docs.microsoft.com/en-us/dotnet/core/diagnostics/distributed-tracing-collection-walkthroughs#sources) into instrumentation.
 
 ```csharp
-var props = Props
-    .FromProducer(() => new MyActor())
-    //this extension enables OpenTracing
-    .WithOpenTracing();
+
+static void ConfigureTracing(WebApplicationBuilder builder) =>
+    builder.Services.AddOpenTelemetryTracing(b =>
+        b.SetResourceBuilder(ResourceBuilder
+                .CreateDefault()
+                .AddService(builder.Configuration["Service:Name"])
+                .AddAttributes(new KeyValuePair<string, object>[]
+                {
+                    new("someCommonTag", builder.Configuration["SomeCommonTag"]),
+                    new("env", builder.Environment.EnvironmentName)
+                })
+            )
+            .AddProtoActorInstrumentation()
+            .AddOtlpExporter(opt => { opt.Endpoint = new Uri(builder.Configuration["Otlp:Endpoint"]); }));
```

Comment:
Later in the document you show to configure Jaeger, but here you use OLTP exporter. Probably this is an error.

Reply:
Jaeger has built an oltp collector, I used it during preparing this instruction. Maybe I could explicitly mention that jaeger supports it: https://www.jaegertracing.io/docs/1.18/opentelemetry/